index.html

```diff
@@ -0,0 +1,1591 @@
+                  <h4 class="funding__heading forth-heading">Truepill</h4>
+                  <div class="funding__price-box">
+                    <h4>$25M</h4>
+                    <h5 class="fifth-heading">Series B</h5>
+                  </div>
+                </a>
+                <!-- The funding block Tag -->
+                <div class="funding__tag-box">
+                  <a href="#" class="funding__tag">
+                    <i class="funding__icon fas fa-tag"></i>
+                    <span>Health Care</span>
+                  </a>
+                </div>
+                <!-- The funding block Bio -->
+                <div class="funding__tag-box">
+                  <a class="funding__bio" href="#">
+                    RacketPal raises £150k in pre-seed round
+                    <span>on&nbsp;UKTN (UK Tech News)</span>
+                  </a>
+                </div>
+              </li>
+              <!-- End Funding Block -->
+
+              <!-- Start Funding Block -->
+              <li class="funding col-m--3">
+                <!-- The funding block header of the section -->
+                <a href="#" class="funding__header">
+                  <img
+                    class="funding__img"
+                    src="https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1280551752593338368%2FAf17dIYK.jpg&signature=d7c37d4f7cda5a5296512fa28f5d94f7"
+                    alt="company logo"
+                  />
+                  <h4 class="funding__heading forth-heading">Truepill</h4>
+                  <div class="funding__price-box">
+                    <h4>$25M</h4>
+                    <h5 class="fifth-heading">Series B</h5>
+                  </div>
+                </a>
+                <!-- The funding block Tag -->
+                <div class="funding__tag-box">
+                  <a href="#" class="funding__tag">
+                    <i class="funding__icon fas fa-tag"></i>
+                    <span>Health Care</span>
+                  </a>
+                </div>
+                <!-- The funding block Bio -->
+                <div class="funding__tag-box">
+                  <a class="funding__bio" href="#">
+                    RacketPal raises £150k in pre-seed round
+                    <span>on&nbsp;UKTN (UK Tech News)</span>
+                  </a>
+                </div>
+              </li>
+              <!-- End Funding Block -->
+            </ul>
+          </section>
+          <!-- End The funding Section -->
+
+          <div class="topic-section s-m-b row">
+            <!-- Start The topic Section -->
+            <section class="topic col-m--2">
+              <!-- Topic Heading -->
+              <h2 class="topic__heading--main second-heading topic-m-b">
+                <a href="#">Hard Fork</a>
+              </h2>
+              <ul class="topic__list">
+                <li class="topic__item">
+                  <article class="topic__article--frist">
```

Comment:
Please reduce the space here 👇
![Annotation 2020-07-10 025647](https://user-images.githubusercontent.com/57726348/87092584-54df0000-c259-11ea-8b4f-fb87f3f78125.jpg)